packages/cli/src/lib/implementation/config-middleware.integration.test.ts

```diff
@@ -25,6 +25,7 @@ describe('configMiddleware', () => {
         config: join(configDirPath, `code-pushup.config.${extension}`),
       });
       expect(config.config).toContain(`code-pushup.config.${extension}`);
+      expect(config.persist.format).toContain(`json`);
```

Comment:
This does not seem related to the test description. If you mean to test that `json` format is always included in the `persist.format`, no matter what configuration is parsed, it deserves a separate test case.

Reply:
introduced options tests in #358 will reuse it here